docs/config.md

```diff
@@ -0,0 +1,135 @@
+- `direct` — Direct (bool). Unused variable. Defaults to true.
+
+### [fuse] FuseConfig
+- `attr_timeout` — AttrTimeout (int). Max timeout for a file system in seconds. Defaults to 0, which sets it to the value of defaultFuseTimeoutSec (1).
+- `entry_timeout` — EntryTimeout (int). TTL for a directory, name lookup in seconds. Defaults to 0, which sets it to the value of defaultFuseTimeoutSec (1).
+- `negative_timeout` — NegativeTimeout (int) Defines overall entry timeout for failed lookups in seconds. Defaults to 0, which sets it to the value of defaultFuseTimeoutSec (1).
+- `log_fuse_operations` — LogFuseOperations (bool). Similar to `debug`, enables debugging for FUSE FS in logs. Please note this often emits sensitive data, so this should be false in production. Defaults to false.
+
+### [background_fetch] BackgroundFetchConfig
+- `disable` — Disable (bool). Disables the background fetcher. Defaults to false.
+- `silence_period_msec` — SilencePeriodMsec (int). Defines time in ms that the background fetcher will be paused when a new image is mounted. Defaults to 0, which sets it to the value of defaultBgSilencePeriodMsec (30000).
+- `fetch_period_msec` — FetchPeriodMsec (int). Specifies how often the fetch will occur in milliseconds. The background fetcher will fetch a single span at this interval. Defaults to 0, which sets it to the value of defaultBgFetchPeriodMsec (500).
+- `max_queue_size` — MaxQueueSize (int). Specifies the max amount of span managers that can be queued. Defaults to 0, which sets it to the value of defaultBgMaxQueueSize (100).
+- `emit_metric_period_sec` — EmitMetricPeriodSec (int). Sets the default amount of interval at which the background fetcher emits metrics. Defaults to 0, which sets it to the value of defaultBgMetricEmitPeriodSec (1).
+
+### [content_store] ContentStoreConfig
+- `type` — Type (string). Sets the content store to use for the snapshotter. Defaults to empty string, which sets it to the value of DefaultContentStoreType ("soci").
+- `namespace` — Namespace (string). Sets the containerd namespace for the snapshotter. Defaults to empty string, which sets it to the value of DefaultContentStoreType ("default").
+
+## config/resolver.go
+
+### [resolver] ResolverConfig
+### [resolver.host] HostConfig
+### [resolver.host.yourmirrorconfigname1.mirrors] MirrorHost
+- `host` — Host (string). Hostname to apply this config to. Defaults to empty string.
+- `insecure` — Insecure (bool). Allows usage of http instead of https only. Defaults to true.
+- `request_timeout_sec` — RequestTimeoutSec (int). Timeout, in seconds, of each request to the registry. Defaults to 0, which creates no timeout.
+
+## config/service.go
+
+### [snapshotter] SnapshotterConfig
+- `min_layer_size` — MinLayerSize (int). Sets the minimum threshold for creating a SOCI index for a layer. Defaults to 0, which sets it to defaultMinLayerSize (10MiB).
```

Comment:
This is the threshold for lazily loading a layer. If the layer is smaller than this value, then the snapshotter will ignore a ztoc for the layer and pull it ahead of time anyway. It also actually defaults to 0 meaning it will never ignore ztocs.